configs/common/FUScheduler.py

```
@@ -113,7 +113,7 @@ class KunminghuScheduler(Scheduler):
             IssuePort(fu=[IntBJU()], rp=[IntRD(4, 1), IntRD(5, 1)])
         ]),
         IssueQue(name='intIQ2', inports=2, size=2*12, oports=[
-            IssuePort(fu=[IntALU()], rp=[IntRD(4, 0), IntRD(5, 0)]),
+            IssuePort(fu=[IntALU(), IntCvt()], rp=[IntRD(4, 0), IntRD(5, 0)]),
```

Ruff (0.14.5)

`IssuePort` may be undefined, or defined from star imports

(F405)